### PR TITLE
8.6 release notes

### PR DESCRIPTION
**Pull request by akeller**
## Description

All epics in implement and validate are included 🚀 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/reference/release-notes/860.md [hunk 1]
```diff
@@ -19,6 +19,71 @@ These release notes identify the new features included in 8.6, including [alpha
...
+
+### Deprecate zbctl and GO client
+
+<!-- https://github.com/camunda/product-hub/issues/2438 -->
+
+With Camunda 8 REST API for Process Management, we will move the Go and zbctl clients to the Camunda Community Hub. This means that these clients won’t be officially supported by Camunda anymore.
+
+With the REST API, the interaction with the Zeebe engine will be much easier; you will be able to work with it in the same way you work with any other REST service. Use the OpenAPI definition, the supported Postman collection, or cURL for an easy getting-started experience and implement API calls in your code. We also recommend giving the new clients and SDKs a try.
+
+If you are interested in maintaining the community projects, contact us by filling out this form in the Camunda Community Hub!
```
Comment (akeller) [inline docs/reference/release-notes/860.md:42]: @aleksander-dytko @xomiamoore does this form exist? How should people contact us?

File: docs/reference/release-notes/860.md [hunk 2]
```diff
@@ -19,6 +19,71 @@ These release notes identify the new features included in 8.6, including [alpha
...
+
+We're thrilled to announce the integration of Zeebe endpoints into Camunda 8 REST API, aimed at enhancing your experience in building process applications. This will ease the installation and onboarding to Camunda 8 as gRPC won't be required anymore. With the introduction of Zeebe endpoints in REST API, we're eliminating the need for gRPC, reducing friction in adoption. Developers can now seamlessly transition from gRPC to REST with the assurance of having the same endpoints, thereby saving time and effort. Use officially supported Camunda SDKs to have an easy migration journey.
+
+### Investigate and resolve incidents with execution listeners
+
+<!-- https://github.com/camunda/product-hub/issues/2103 -->
+
+Users can now easily check details of active and completed execution listeners for process activities and process instances. Operations engineers can efficiently resolve execution listener incidents. This enhancement streamlines operations and ensures smoother handling of incidents, ultimately improving time to unblock process execution.
+
+### Deprecate zbctl and GO client
```
Comment (akeller) [inline docs/reference/release-notes/860.md:34]: @aleksander-dytko should we link to https://camunda.com/blog/2024/09/deprecating-zbctl-and-go-clients/?

File: docs/reference/release-notes/860.md (hunk 1 above)
Comment (xomiamoore) [reply docs/reference/release-notes/860.md:42]: It's the "Maintainer Proposal" template here -- https://github.com/Camunda-Community-Hub/community/issues/new/choose

**Review by mesellings — DISMISSED** [review]
Lgtm 👍 

I've committed edits as follows:

- Sorted items alphabetically.
- Reworded for conciseness, clarity, front-loading, grammar etc.

**Comment by mesellings** [comment]
@akeller I have one item now needed to add in (SaaS uptime work), please do not merge until I have added it 👍

**Review by mesellings — DISMISSED** [review]
Lgtm 👍

**Comment by mesellings** [comment]
@akeller I have added in the new item for https://github.com/camunda/product-hub/issues/2377 and re-approved 👍

**Comment by mesellings** [comment]
@akeller One more late addition, just adding it now (encryption keys)

**Comment by mesellings** [comment]
Added in item for https://github.com/camunda/issues/issues/801  👍

File: docs/reference/release-notes/860.md [hunk 3]
```diff
@@ -19,6 +19,137 @@ These release notes identify the new features included in 8.6, including [alpha
 | ---------------------- | ---------------------------- | ------------ | ------------ | --------------------------------------------------------------------------------------- |
 | 8 October 2024         | 14 April 2026                |              |              | [Self-Managed update guide](self-managed/operational-guides/update-guide/850-to-860.md) |
 
+<!-- https://github.com/camunda/product-hub/issues/2377 -->
+
+### Advanced SaaS offering <span class="badge badge--long" title="This feature affects SaaS">SaaS</span> <span class="badge badge--long" title="This feature affects Console">Console</span>
```
Comment (akeller) [inline docs/reference/release-notes/860.md:24]: Nitpick - can you add the URL under the heading?

File: docs/reference/release-notes/860.md [hunk 4]
```diff
@@ -19,6 +19,137 @@ These release notes identify the new features included in 8.6, including [alpha
...
+Interested in maintaining these community projects? Contact us by completing the [Maintainer Proposal form](https://github.com/Camunda-Community-Hub/community/issues/new/choose) in the Camunda Community Hub.
+:::
+
+### Email Connector <span class="badge badge--long" title="This feature affects Connectors">Connectors</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2430 -->
+
+The new Email Connector allows you to integrate your BPMN service with any email server using POP3, IMAP, or SMTP. You can use this Connector to automate the retrieval, deletion, search, and organization of emails directly within your processes.
+
+<!-- https://github.com/camunda/issues/issues/801 -->
```
Comment (akeller) [inline docs/reference/release-notes/860.md:89]: Same nitpick, under the heading. Also, was there no corresponding PDP issue for this in the Product Hub?

**Comment by akeller** [comment]
Thanks @mesellings - we can define a process for review, ordering, etc. ahead of the next minor release. I'm fine with alphabetical, but we should check in with PM. I usually just copy over as I see them in the Product Hub and then do a final review right before merging.

File: docs/reference/release-notes/860.md (hunk 3 above)
Comment (akeller) [reply docs/reference/release-notes/860.md]: fixed

**Comment by akeller** [comment]
Adding back the TW group for review in case Mark is already offline for the day.

**Review by mesellings — APPROVED** [review]
@akeller Lgtm 👍

File: docs/reference/release-notes/860.md (hunk 3 above)
Comment (mesellings) [reply docs/reference/release-notes/860.md]: Thank you - apologies!

File: docs/reference/release-notes/860.md (hunk 4 above)
Comment (mesellings) [reply docs/reference/release-notes/860.md]: There was one after all: https://github.com/camunda/product-hub/issues/1218 - do you want me to correct this in a PR after the release?

Comment (mesellings) [reply docs/reference/release-notes/860.md]: Will do 👍

**Pull request merged**